itemtypes.md

```diff
@@ -0,0 +1,40 @@
+# Item Type Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint is currently still named entitytypes, but may be renamed still.
```

Comment:
I think we should rename these endpoints to be `itemtypes` immediately.  You already have this in a file called `itemtypes.md`, so I see no reason to keep the endpoint descriptions as "entitytypes"

Reply:
I've renamed it now, the links to our test server will not work at the moment, but I made sure all examples are included inline